docs/sources/alerting/alerting-rules/create-mimir-loki-managed-rule.md

```diff
@@ -3,6 +3,7 @@ aliases:
   - /docs/grafana/latest/alerting/alerting-rules/create-mimir-loki-managed-rule/
   - /docs/grafana/latest/alerting/unified-alerting/alerting-rules/create-cortex-loki-managed-recording-rule/
   - /docs/grafana/latest/alerting/unified-alerting/alerting-rules/create-mimir-loki-managed-recording-rule/
+  - /docs/grafana/latest/alerting/unified-alerting/alerting-rules/create-mimir-loki-managed-rule
```

Comment:
No `/` at the end?

Reply:
oops thanks!
I'm not sure if it makes a difference-but since the others do, I will add.